app/views/posts/index.html.erb

```diff
@@ -1 +1,20 @@
-<h1>Posts here</h1>
\ No newline at end of file
+<%= render 'shared/user_profile', user: @user %>
+
+<% @posts.each  do  |post|%>
+<div>
+    <a href="/users/<%= post.author_id%>/posts/<%= post.id %>"><%= post.title%></a>
+        <p><%= post.text%></p>
+        <p>
+            Comments: <%= post.comments_counter %> Likes: <%= post.likes_counter%>
+        </p>
+</div>
+<div>
+<% Post.recent_comments(post).each  do |comment|%>
+    <p><%= comment.author.name%> - <%= comment.text %></p>
+    <%end%>
+</div>
+<br />
+<%end%>
+<div id="bottom-button">
+<button>Pagination</button>
+</div>
```

Comment:
- [ ] When I navigate to the post page of a user posts page I get an error
![Screenshot from 2023-07-20 22-25-59](https://github.com/joyapisi/blog_app/assets/66097371/6ecbadb9-4d10-4fd3-9c7a-672e04ccf452)